network_diffusion/tpn/tpnetwork.py

```diff
@@ -0,0 +1,62 @@
+"""A script where a temporal network is defined."""
+
+from typing import Any, Dict, List, Optional  # , Union
+
+import networkx as nx
+
+from network_diffusion.utils import read_tpn
+
+
+class TemporalNetwork:
+    """Container for a temporal network."""
+
+    # def __init__(self, snaps: Dict[Union[str, int], nx.Graph]) -> None:
+    def __init__(self, snaps: Dict[int, nx.Graph]) -> None:
+        """
+        Create a temporal network object.
+
+        :param snaps: snapshots of the temporal network.
+        """
+        self.snaps = snaps
+
+    @classmethod
+    def from_txt(
+        cls, file_path: str, time_window: int, directed: bool = True
+    ) -> "TemporalNetwork":
+        """
+        Load a temporal network from a txt file.
+
+        Note, the txt file should be in the form of: SRC DST UNIXTS.
+        The timestamps UNIXTS should be sorted.
+
+        :param file_path: path to the file
+        :param time_window: the time window size for each snapshot
```

Comment:
what's a unit of the time window size? Seconds, minutes, hours? If my comment is irrevelant (I'm not an expert in the area of temporal networks, then please let me know :D ) 

Ayway, if i'm right then I'd update a docstring

Reply:
Typically, the unit is second. I have updated the docstring and committed to the repository.